modules/entity/src/models.ts

```diff
@@ -78,12 +78,12 @@ export interface EntityStateAdapter<T> {
   map<S extends EntityState<T>>(map: EntityMap<T>, state: S): S;
 }
 
-export interface EntitySelectors<T, V> {
+export type EntitySelectors<T, V> = {
```

Comment:
Could we add a test for the current compilation error?
That way we can't easily revert it in the future.